File: exercicios/R/1065/1065.r
input = file('stdin','r')

qt <-c(1:5)

count = as.integer(0)

for (i in qt) {
   v = as.integer(readLines(input,n=1))

   if (v%%2==0) {
      count = count + 1
   }
}
write(sprintf("%d valores pares",count),'')

File: exercicios/JavaScript/1065/1065.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var count = 0;

for(let i=0; i<5; i++){
    var valor = parseInt(lines.shift())

    if(valor % 2 === 0){
        count +=1;
    }
}
console.log(count + " valores pares")